graphx/src/main/scala/org/apache/spark/graphx/VertexRDD.scala

```diff
@@ -122,6 +122,14 @@ abstract class VertexRDD[VD](
   def mapValues[VD2: ClassTag](f: (VertexId, VD) => VD2): VertexRDD[VD2]
 
   /**
+   * For each VertexId present in both `this` and `other`, minus will act as a set difference
+   * operation returning only those unique VertexId's present in `this`.
+   *
+   * @param other an RDD to run the set operation against
+   */
+  def minus(other: RDD[(VertexId, VD)]): VertexRDD[VD]
```

Comment:
It would be good to add a version that takes a VertexRDD, similar to the other methods below. That way we could avoid calling createUsingIndex when we already have two VertexRDDs that share the same index.